eipw-lint/tests/lint_markdown_relative_links.rs

```diff
@@ -193,3 +193,31 @@ https://example.com/
 "#
     );
 }
+
+#[tokio::test]
+async fn anchor_link() {
+    let src = r#"---
+header: value1
+---
+
+<a href="https://example.com/">example</a>
```

Comment:
Could you add an additional test for:

```html
<a href="//example.com/">example</a>
```

Reply:
Are you expecting this case to throw an error? From what I can tell this shouldn't because it doesn't get caught by the `://` regex for relative links.

Reply:
I don't think `://example.com` is valid, while `//example.com` is a [network-path reference](https://www.rfc-editor.org/rfc/rfc3986#section-4.2) (commonly known as a protocol-relative URL).

We don't want to allow protocol-relative URLs.

Reply:
Fixed and amended it within the previous commit